Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Tooltips/TooltipEnums/ConnectorFollowType.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.UX.ToolTips
+{
+    /// <summary>
+    /// How does the Tooltip track with its parent object
+    /// </summary>
+    [Flags]
+    public enum ConnectorFollowType
+    {
+        /// <summary>
+        /// The anchor will follow the target - pivot remains unaffected
+        /// </summary>
+        AnchorOnly = 0x0,
```

Comment:
Please consider replacing hex values with shift notation as per the style guide.

Reply:
Could I have a link to the style guide?

Reply:
https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/CodingGuidelines.md